tick/optim/model/src/coxreg_partial_lik.h

```diff
@@ -9,8 +9,7 @@
 
 #include "model.h"
 
-
-class ModelCoxRegPartialLik : public Model {
+class DLL_PUBLIC ModelCoxRegPartialLik : public Model {
```

Comment:
Oh that's true, I have no idea why they appear there but that's another problem!